src/main/java/teammates/storage/api/CommentsDb.java

```diff
@@ -340,6 +341,18 @@ private void updateInstructorEmailAsGiver(String courseId, String oldInstrEmail,
         getPm().close();
     }
     
+    private void updateInstructorEmailAsLastEditor(String courseId, String oldInstrEmail,
+                                                   String updatedInstrEmail) {
+        List<Comment> lastEditorComments = getCommentEntitiesForLastEditor(courseId, oldInstrEmail);
+        
+        for (Comment lastEditorComment : lastEditorComments) {
+            lastEditorComment.setLastEditorEmail(updatedInstrEmail);
+        }
+        
+        log.info(Const.SystemParams.COURSE_BACKUP_LOG_MSG + courseId);
```

Comment:
Still think this is a redundant log. It should at least identify the comment to be of any use.